core/platform/models.py

```diff
@@ -84,21 +96,47 @@ def import_models(cls, model_names):
 
     @classmethod
     def import_transaction_services(cls):
+        """Imports and returns gae_transaction_services.
+
+        Returns:
+            module. The gae_transaction_services modue.
+        """
         from core.platform.transactions import gae_transaction_services
         return gae_transaction_services
 
     @classmethod
     def import_current_user_services(cls):
+        """Imports and returns gae_current_user_services.
+
+        Returns:
+            module. The gae_current_user_services module.
+        """
         from core.platform.users import gae_current_user_services
         return gae_current_user_services
 
     @classmethod
     def import_app_identity_services(cls):
+        """Imports and returns gae_app_identity_services.
+
+        Returns:
+            module. The gae_app_identity_services module.
+        """
         from core.platform.app_identity import gae_app_identity_services
         return gae_app_identity_services
 
     @classmethod
     def import_email_services(cls):
+        """Imports and returns the email services module
+        specified in feconf.py.
+
+        Returns:
+            module. The email_services module to use,
+            based on the feconf.py setting.
+
+        Raises:
+            Exception: Exception: feconf.EMAIL_SERVICE_PROVIDER does not
```

Comment:
"Exception: " is repeated. Please proofread before requesting review -- it's useful to look at the files as they [are shown on GitHub](https://github.com/oppia/oppia/pull/3226/files).